addons/repair/views/repair_views.xml

```diff
@@ -3,6 +3,25 @@
 
     <data>
 
+    <record id="repair_order_view_activity" model="ir.ui.view">
+        <field name="name">repair.order.activity</field>
+        <field name="model">repair.order</field>
+        <field name="arch" type="xml">
+            <activity string="Activity view">
+                <templates>
+                    <div t-name="activity-box">
+                        <field name="user_id" widget="many2one_avatar_user" domain="[('share', '=', False)]"/>
```

Comment:
Shouldn't we put that domain directly on the field actually ? Activities should not be assigned to portal users anyway ...